gateway_to_backend/README.md

```diff
@@ -173,12 +178,18 @@ backends. It also allows the backends to monitor the gateways status.
 > **content:**
 > [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[StatusEvent][message_StatusEvent]
 
-The message must have the retained flag set to true to allow a new
+The message must have the retain flag set to true to allow a new
 backend to immediately be noticed of all the available gateways when
 subscribing to this topic.
 
 Once connected to the MQTT broker, status must be set to *ONLINE*.
 
+As some broker do not support the retain flag, sending periodically the status
+of the gateway can be implemented instead of only once with the retained flag.
+Consequently, backends will have to wait for a period time after the subscribing to
+this topic before being noticed of all gateway status.
+A period of 30s is a good tradeoff between loading the broker and reactivity for backends.
```

Comment:
30 seconds period might be too dense if backend needs to handle thousands of gateways. Maybe 30s - 5min would be better range. This should be configurable value.

Reply:
Why can we have it as:
* backends will pull it when needed
* GW will publish it on change

Reply:
That would change the API, not just the implementation